ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -160,7 +161,7 @@ export const NotebookResourceCard = fp.flow(
         fromWorkspaceNamespace={this.props.resource.workspaceNamespace}
         fromWorkspaceName={this.props.resource.workspaceFirecloudName}
         fromResourceName={dropNotebookFileSuffix(this.props.resource.notebook.name)}
-        resourceType={ResourceType.NOTEBOOK}
+        resourceType={this.resourceType}
```

Comment:
Do you happen to know why this was hard-coded before?

Reply:
I think it was hardcoded since it's specifically a `notebook-resource-card` and so the resource type in this case is always Notebook.

Reply:
It's currently hardcoded to `Notebook`, but `ResourceType.NOTEBOOK` is set to `notebook`. Do you know if there is any behavior that depends on this?

Reply:
as of this PR (merged today) those should all be referring to ResourceType.NOTEBOOK which has the value `Notebook` https://github.com/all-of-us/workbench/pull/2671/files so we should be covered in the behavior here.

Reply:
Sweet, I think my `master` is just behind then